pandas/tests/frame/test_arithmetic.py

```diff
@@ -127,132 +126,97 @@ def test_df_add_flex_filled_mixed_dtypes(self):
              'B': ser * 2})
         tm.assert_frame_equal(result, expected)
 
-    def test_arith_flex_frame(self):
-        seriesd = tm.getSeriesData()
-        frame = pd.DataFrame(seriesd).copy()
+    @pytest.mark.parametrize('op', ['add', 'sub', 'mul', 'div', 'truediv',
```

Comment:
can you change these to use the fixture: all_arithmetic_operators

Reply:
will do